sql/src/test/java/io/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -122,6 +122,14 @@ public int getMaxTopNLimit()
       return 0;
     }
   };
+  private static final PlannerConfig PLANNER_CONFIG_NO_HLL = new PlannerConfig()
```

Comment:
There's also PLANNER_CONFIG_NO_TOPN to disable topn approximations, so I want to keep them different for that reason.